### DISCUSSION TITLE
Aggregations with Security Onion

### DISCUSSION THREAD

**Discussion by idolaman** [Security-Onion-Solutions/securityonion#3041] [Unsupported Versions] [answered]
Hello,
Is it possible to run aggregations with sigma \ elastalert in security onion?
for example - alert when there are three or more failed authentications to windows computer (wineventlog) in explicit time.

If the answer is no, there is planning for this kind of alerts?
Thanks :)

**Comment by stevewillia** [comment, marked as answer]
A frequency rule should handle that

index: "*:so-ids-*"

# (Required)
# Type of alert.

type: frequency

# (Required, change specific)
# The field to look for changes in
num_events: 3
timeframe:
     minutes: 5

filter:
    - query:
       query_string:
        query: "event.severity: 3"
        query: "event.dataset: alert"


Just build your query on the events you want included

Steve
